mantidproject/mantid
# Indirect - MSDFit - Display and update properties in property table

In the MSDFit Indirect Data Analysis interface, when the model is updated, the property table does not display the properties for the current model (e.g. Intensity and MSD for a Gaussian model).
These properties should be displayed and updated and the plot preview should properly display the results of a fit.

**To test:**
1. Navigate to Interfaces > Indirect > Data Analysis > MSDFit.
2. Select any model.
3. In the input field enter the supplied reduced file (see below).
4. Ensure the spectra range covers the entire input.
5. Click 'Run'.
6. Scroll through all spectrum, using the 'Plot Spectrum' spinner and check that the sample and fit are plotted for each and the values in the property table are updated and correspond to the created \*\_Parameters workspace.
7. Repeat steps 5 and 6 but with a sub-range for the spectra (doesn't cover entire range), also check that spectrum outside the fitted range plot the sample but not a fit.
8. Repeat step 6, but fit a single spectrum using 'Fit Single Spectrum' instead of 'Run'.

**Test File**
[irs26176_graphite002_red.zip](https://github.com/mantidproject/mantid/files/1341666/irs26176_graphite002_red.zip)



Fixes #20642. 

**Release Notes** 


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
